content/05-mysql/12-short-guides/03-exporting-schemas.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: 'How to export database and table schemas in MySQL'
+metaTitle: "Export schemas for MySQL databases and tables"
+metaDescription: "Learn how to export your database objects from MySQL for analysis, migrating, backups, and more."
+metaImage: '/social/generic-mysql.png'
+authors: ['justinellingwood']
+---
+
+## Introduction
+
+In relational databases, the [database schema](/intro/database-glossary#schema) defines the structure of the database and it's component parts like tables, fields, and indexes.  Extracting and exporting this information is useful in many scenarios including backups, migrating to new environments, visualizing data structures, and managing these structures within a codebase.
+
+In this short guide, we'll discuss how to export MySQL database schemas using the `mysqldump` command.  While this utility can export many types of data from MySQL, we'll focus on extracting the data structures themselves in this guide.
+
+<PrismaOutlinks>
+
+If you are using Prisma, you can connect with your MySQL server and manage your data using the [MySQL database connector](https://www.prisma.io/docs/concepts/database-connectors/mysql#connection-details).
+
+</PrismaOutlinks>
+
+## Basic usage
+
+The basic command needed to export the database schema from MySQL looks like this:
+
+```shell
+mysqldump --user=USERNAME --host=HOSTNAME --password --no-data DATABASE > schema.sql
+```
+
+The options here can be divided into two separate categories.
+
+The first category defines the generic basic connection information that you need to provide in order to connect with any MySQL utility:
+
+* `--user=` / `-u`: The database username that you want to authenticate with
+* `--password` / `-p`: Force `mysqldump` to prompt for a password to authenticate
+* `--host=` / `-h`: The hostname or IP address of where MySQL is located
+* `--port=` / `-p`: The port number where MySQL is listening
+
+If you are connecting to a local MySQL instance running in the default configuration, you can typically omit the host and port options.
+
+The second category tells `mysqldump` what to export:
+
+* `--no-data` / `-d`: This tells the utility to only export the structure itself, not the records they contain.
```

Comment:
```suggestion
* `--no-data` / `-d`: This tells the utility to only export the structure itself, not the records they contain
```

Reply:
Not sure if this statement is considered differently than those above regarding end-of-sentence punctuation.

Reply:
It is "more" of a complete sentence, so maybe the period should stay. Just pointing it out in case.